interop/kotlin/bootstrap.sh

```diff
@@ -23,32 +24,32 @@ jna_architectures=(
   "linux-x86-64"
   "win32-x86-64"
 )
-target_triples=(
+ret_names=(
   "aarch64-apple-darwin"
   "x86_64-apple-darwin"
   "aarch64-unknown-linux-gnu"
   "x86_64-unknown-linux-gnu"
   "x86_64-pc-windows-gnu"
 )
-file_names=(
-  "libuniffi_radix_engine_toolkit_uniffi.dylib"
-  "libuniffi_radix_engine_toolkit_uniffi.dylib"
-  "libuniffi_radix_engine_toolkit_uniffi.so"
-  "libuniffi_radix_engine_toolkit_uniffi.so"
-  "radix_engine_toolkit_uniffi.dll"
+suffixes=(
+  "dylib"
+  "dylib"
+  "so"
+  "so"
+  "dll"
 )
 
-
 for (( i=0; i<${#jna_architectures[@]}; i++ ));
 do
   arch_name=${jna_architectures[$i]}
-  target_triple=${target_triples[$i]}
-  file_name=${file_names[$i]}
+  ret_name=${ret_names[$i]}
+  suffix=${suffixes[$i]}
 
   echo "Extracting for architecture $arch_name"
 
   mkdir $res/"$arch_name"
-  mv $artifacts/$crate_name-$target_triple/$file_name $jni/"$arch_name"/$file_name
+  mv $artifacts/"$crate_name"-"$ret_name"/*."$suffix" $res/"$arch_name"/libuniffi_radix_engine_toolkit_uniffi."$suffix"
```

Comment:
I believe that this bit works for Linux and MacOS, but doesn't work for Windows. On Windows, the canonical name of libraries should not have the `lib` prefix, this is why I changed `suffixes` to  `file_names`